ci-operator/step-registry/ipi/deprovision/deprovision/ipi-deprovision-deprovision-commands.sh

```diff
@@ -105,6 +105,10 @@ if test -f "${SHARED_DIR}/proxy-conf.sh"; then
     # when bastion host is provisioned in cluster resource group, once the bastion is destroyed in the destroy process,
     # the running destroy process would be interrupted and failed. E.g: azure-ipi-public-to-private jobs.
     echo "proxy-conf.sh detected, but not required by azure4 clusters while destroying cluster, skip proxy setting"
+  elif [[ "${CLUSTER_TYPE}" == nutanix ]]; then
```

Comment:
The whole if block is for deciding export proxy or not, Your case is for deciding `SSL_CERT_FILE` or not, I think it need a separate if block.